schema.h

```diff
@@ -291,6 +291,26 @@ struct Card : yoda::Padawan {
   }
 };
 
+struct CardAuthor : yoda::Padawan {
```

Comment:
Totally! I thought of it for a couple seconds, looks like the best would be:

1. Have the functionality ready,
2. Have the event replay logic ready, and then
3. Change DB schema, both here and in another place where we're missing the identifier.

Liveable, or am I missing something important? :-)